### PR TITLE
Fix wrong var name in NOTES when Ingress is enabled

### PR DESCRIPTION
**Pull request by komljen**
*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

File: stable/wordpress/templates/NOTES.txt [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
   You should be able to access your new WordPress installation through
 
-  http://{{- .Values.ingress.hostname }}/admin
+  {{- range .Values.ingress.hosts }}
+  http://{{ .name }}/admin
```
Comment (mattfarina) [inline stable/wordpress/templates/NOTES.txt]: It could be https instead of http if TLS is enabled. Could we handle that case as well?

Comment (unguiculus) [reply stable/wordpress/templates/NOTES.txt]: Here's an example:

https://github.com/kubernetes/charts/blob/master/incubator/keycloak/templates/NOTES.txt#L12-L14

Comment (komljen) [reply stable/wordpress/templates/NOTES.txt]: Yeah, thanks for the suggestion. I added that also.

**Pull request merged**